### PR TITLE
Adding support for opensuse

### PR DESCRIPTION
**Pull request by davidjumani**

### CODE REVIEW

File: Ansible/roles/cloudmonkey/tasks/suse.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+
+#Copyright 2016 ShapeBlue
```
Comment (rhtyd) [inline Ansible/roles/cloudmonkey/tasks/suse.yml:3]: nit picking - year

File: Ansible/roles/cloudstack-manager/tasks/main.yml [hunk 2]
```diff
@@ -32,19 +32,32 @@
 - name: now i'm connected, get facts
   os_facts:
 
+# For some weird reason, it thinks SUSE is Ubuntu
+- name: Check if SUSE
+  shell: cat /etc/os-release | grep -i suse
+  register: result
+  when: ansible_distribution == 'Ubuntu'
```
Comment (rhtyd) [inline Ansible/roles/cloudstack-manager/tasks/main.yml:39]: could be due to use of older ansible version

File: Ansible/roles/cloudstack-manager/tasks/suse-acs.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+
+#Copyright 2016 ShapeBlue
```
Comment (rhtyd) [inline Ansible/roles/cloudstack-manager/tasks/suse-acs.yml:3]: nit picking - year

File: Ansible/roles/cloudstack-manager/tasks/suse.yml [hunk 4]
```diff
@@ -0,0 +1,87 @@
+---
```
Comment (rhtyd) [inline Ansible/roles/cloudstack-manager/tasks/suse.yml:1]: nit - No license here

File: Ansible/roles/kvm/tasks/kvm_networking_bridge_suse.yml [hunk 5]
```diff
@@ -0,0 +1,35 @@
+---
+
+#Copyright 2016 ShapeBlue
```
Comment (rhtyd) [inline Ansible/roles/kvm/tasks/kvm_networking_bridge_suse.yml:3]: nit picking - year

File: Ansible/roles/kvm/tasks/kvm_networking_ovs_suse.yml [hunk 6]
```diff
@@ -0,0 +1,99 @@
+---
+
+#Copyright 2016 ShapeBlue
```
Comment (rhtyd) [inline Ansible/roles/kvm/tasks/kvm_networking_ovs_suse.yml:3]: nit picking - year

File: Ansible/roles/kvm/tasks/suse-acs.yml [hunk 7]
```diff
@@ -0,0 +1,45 @@
+---
+
+#Copyright 2016 ShapeBlue
```
Comment (rhtyd) [inline Ansible/roles/kvm/tasks/suse-acs.yml:3]: nit picking - year

File: Ansible/roles/kvm/tasks/suse.yml [hunk 8]
```diff
@@ -0,0 +1,171 @@
+---
```
Comment (rhtyd) [inline Ansible/roles/kvm/tasks/suse.yml:1]: No license

**Review by rhtyd — APPROVED** [review]
LGTM, minor commits:
nit picking - year wrong/old in license headers or headers missing

File: Ansible/roles/cloudstack-manager/tasks/setup-mgmt.yml [hunk 9]
```diff
@@ -50,7 +50,11 @@
   tags:
     - setup_mgmt
 
-
+- name: Setup CloudStack Manager (SUSE)
+  shell: sleep {{ play_hosts.index(inventory_hostname) | int * 15 }} && /usr/bin/cloudstack-setup-management
+  when: ansible_distribution == "SUSE"
+  tags:
+    - setup_mgmt
```
Comment (DaanHoogland) [inline Ansible/roles/cloudstack-manager/tasks/setup-mgmt.yml:57]: why do we need a separate task for these? a lot of them are only different in conditions and those can be combined.

**Review by DaanHoogland — CHANGES_REQUESTED** [review]
looks generally good,
about the copy rights, don't remove the older year unless you checked with a lawyer. you don't have to put it in newer files but there is something about removing it, even yourself.
do we need 3rd party testing?

**Comment by andrijapanicsb** [comment]
Was this tested on our Jenkins, before merging @DaanHoogland @davidjumani?

**Comment by davidjumani** [comment]
Ofcourse @andrijapanicsb :boom:

**Pull request merged**